### ISSUE TITLE
Not-so-personal Data; including Gender, Age, Bio, and others

### ISSUE THREAD

**Issue by dsevillamartin** [dav-is/flarum-ext-socialprofile#12]
Let user choose their gender, for other user's reference.
Isn't required, though :smile:

**Comment by dav-is** [comment]
Along with that would probably be age, location, and your bio (as it will be removed from core soon)

**Comment by dsevillamartin** [comment]
Yeah, so like optional not-so-personal data, for the benefit of others talking with or to you
